dts/arm/st/wb0/stm32wb0.dtsi

```diff
@@ -217,6 +217,22 @@
 			interrupts = <7 0>;
 			status = "disabled";
 		};
+
+		adc1: adc@41006000 {
+			compatible = "st,stm32wb0-adc";
+			reg = <0x41006000 256>;
+			/* On STM32WB0, the two ADC clock domains (ANALOG / DIGITAL)
+			 * can be controlled independently. Expose this feature to
+			 * the driver by having two `clocks` property entries:
+			 *  - first entry is digital part of ADC block (always-on)
+			 *  - second entry is analog part of ADC block (on-demand)
+			 */
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 (1 << 4)>,
+				 <&rcc STM32_CLOCK_BUS_APB1 (1 << 5)>;
```

Comment:
Now that #79683 has been merged, this node should be updated

Reply:
Done. Also added a commit to cleanup the missed nodes (I2C/SPI).